src/routes/volume/VolumeActions.js

```diff
@@ -5,6 +5,41 @@ import { DropOption } from '../../components'
 const confirm = Modal.confirm
 
 function actions({ selected, engineImages, showAttachHost, detach, showEngineUpgrade, deleteVolume, showBackups, showSalvage, rollback, showUpdateReplicaCount, showExpansionVolumeSizeModal, showCancelExpansionModal, createPVAndPVC, changeVolume, confirmDetachWithWorkload, commandKeyDown }) {
+  const deleteWranElement = (record) => {
+    let workloadResources = ''
+    let hasPvTooltipText = ''
+    let pvResources = ''
+
+    if (record && record.kubernetesStatus && record.kubernetesStatus.pvStatus) {
+      hasPvTooltipText = (<div>{`The following resources associated with this volume (${record.name}) will be deleted:`}</div>)
+      pvResources = (<div style={{ margin: '10px', color: '#b7b7b7' }}>
+        <div style={{ paddingLeft: '10px' }}>PV Name: {record.kubernetesStatus.pvName}</div>
+        <div style={{ paddingLeft: '10px' }}>PVC Name: {record.kubernetesStatus.pvcName}</div>
```

Comment:
Is it possible to only show `PVC Name` if there is a `PersistentVolumeClaim`?

`PVC Name` should also only be shown if `lastPVCRefAt` is empty.